Node/vue-browser/public/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow:
```

Comment:
This can be removed if we aren't using it

Reply:
done